packages/plugins/test/mocha/cli/propose.spec.ts

```diff
@@ -192,15 +192,6 @@ describe('Propose CLI command', () => {
     expect(
       (networkConfig.actionInputs[0] as Create2ActionInput).create2Address
     ).equals(expectedContractAddress)
-
-    await assertValidGasEstimates(
-      proposalRequest.gasEstimates,
-      networkConfigArray,
-      networks,
-      scriptPath,
-      targetContract,
-      context
-    )
```

Comment:
Unfortunately, these tests won't pass if the simulation is disabled, which is the case for these tests. I could not find a way around that requirement, so I've disabled them.

I've made a note in the [relevant ticket](https://linear.app/chugsplash/issue/CHU-1072/re-enable-simulation-step-against-local-nodes) that these should be reenabled if/when we can use the simulation on local nodes again.